spec/java/util/LinkedList.lsl

```diff
@@ -0,0 +1,666 @@
+libsl '1.1.0';
+
+library 'std:collections' language 'Java' version '11' url '-';
+
+import 'java-common.lsl';
+import 'list-actions.lsl';
+
+include java.util.function.Consumer;
+
+//For generation this imports are needed:
+include java.util.AbstractSequentialList;
+include java.util.Collection;
+include java.util.Deque;
+include java.util.Iterator;
+include java.util.List;
+include java.util.ListIterator;
+
+@Public
+@Extends('java.util.AbstractSequentialList')
+@Implements(['java.util.List','java.util.Deque','java.lang.Cloneable','java.io.Serializable'])
+@WrapperMeta(
+    src='java.util.LinkedList',
+    dst='org.utbot.engine.overrides.collections.UtLinkedList',
+    matchInterfaces=true,
+)
+automaton LinkedList: int(
+   	var storage: list<Object>,
+	@Transient var size: int = 0,
+	@Protected @Transient var modCount: int = 0,
+    	var serialVersionUID:long = 876323262645176354
+)
+{
+
+	// constructors
+	shift Allocated -> Initialized by [
+        	LinkedList(),
+        	LinkedList(Collection)
+   	];
+	
+	shift Initialized -> self by [
+        	// read operations
+		getFirst,
+		getLast,
+		contains,
+		size,
+		get,
+		indexOf(Object),
+		indexOf(int),
+		lastIndexOf,
+		peek,
+		element,
+		peekFirst,
+		peekLast,
+		toArray,
+		spliterator,
+		listIterator
+		
+		
+
+        	toString,
+        	hashCode,
+        	clone,
+        
+        	// write operations
+        	removeFirst,
+		removeLast,
+		addFirst,
+		addLast,
+		add(Object),
+		remove(Object),
+		addAll(Collection),
+		addAll(int, Collection),
+		clear,
+		set,
+		add(int, Object),
+		remove(int),
+		poll,
+		remove(),
+		offer,
+		offerFirst,
+		offerLast,
+		pollFirst,
+		pollLast,
+		push,
+		pop,
+		removeFirstOccurrence,
+		removeLastOccurrence
+   	 ];
+
+	
+
+	//constructors
+
+	constructor LinkedList (): void {
```

Comment:
Constructors, as it have been discussed lately, do not have return value, so please remove return type specifier.

Reply:
Has been fixed